Lib/enum.py

```diff
@@ -744,11 +742,10 @@ def __xor__(self, other):
 
     def __invert__(self):
         members, uncovered = _decompose(self.__class__, self._value_)
-        inverted_members = [
-                m for m in self.__class__
-                if m not in members and not m._value_ & self._value_
-                ]
-        inverted = reduce(_or_, inverted_members, self.__class__(0))
+        inverted = self.__class__(0)
+        for m in self.__class__:
+            if m not in members and not (m._value_ & self._value_):
+                inverted = inverted | m
```

Comment:
I don't see how this change is related to removing imports.

Reply:
`functools.reduce` no longer imported.